CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # CHANGELOG
 
+
+## v1.0.0 - 02-28-2022
```

Comment:
is this release in the past? 😄

Reply:
I was hoping to release it on 28th Feb :) I'll update it to today.